src/mlpack/tests/recurrent_network_test.cpp

```diff
@@ -1413,4 +1413,69 @@ BOOST_AUTO_TEST_CASE(BRNNTrainReturnObjective)
   BOOST_REQUIRE_EQUAL(std::isfinite(objVal), true);
 }
 
+/**
+ * Test that RNN::Train() does not give an error for large rho.
+ */
+BOOST_AUTO_TEST_CASE(LargeRhoValueRnnTest)
+{
+  // Setting rho value greater than sequence length which is 17.
+  const size_t rho = 100;
+  const size_t hiddenSize = 128;
+  const size_t numLetters = 256;
+  using MatType = arma::cube;
+  std::vector<std::string> trainingData;
+  trainingData.push_back(std::string("THIS IS THE INPUT 0"));
+  trainingData.push_back(std::string("THIS IS THE INPUT 1"));
+  trainingData.push_back(std::string("THIS IS THE INPUT 2"));
```

Comment:
No need of push_back 
```
std::vector<std::string>trainingData = { "THIS IS THE INPUT 0", "THIS IS THE INPUT 1",
 "THIS IS THE INPUT 3" } ;
```

why are you writing std:string(""THIS IS THE INPUT 0"), isn't `""` already a string ?
also if it isn't changing anywhere we can use const